### PR TITLE
Demonstrate modification of the IMAAdsRequest.

### PR DESCRIPTION
**Pull request by timothyrodgers**

### CODE REVIEW

File: IMA/BasicIMAPlayer/objc/BasicIMAPlayer/ViewController.m [hunk 1]
```diff
@@ -274,7 +279,17 @@ - (void)playbackController:(id<BCOVPlaybackController>)controller playbackSessio
...
 
-#pragma mark IMAWebOpenerDelegate Methods
+#pragma mark - IMAPlaybackSessionDelegate Methods
+
+- (void)willCallIMAAdsLoaderRequestAdsWithRequest:(IMAAdsRequest *)adsRequest forPosition:(NSTimeInterval)position
+{
+    // for demo purposes, increase the VAST ad load timeout.
+    adsRequest.vastLoadTimeout = 3000.;
+    NSLog(@"ViewController Debug - IMAAdsRequest.vastLoadTimeout set to %.1f milliseconds.", adsRequest.vastLoadTimeout);
+
```
Comment (sjbushell) [inline IMA/BasicIMAPlayer/objc/BasicIMAPlayer/ViewController.m]: extra line?

File: IMA/FairPlayIMAPlayer/objc/FairPlayIMAPlayer/ViewController.m [hunk 2]
```diff
@@ -305,7 +310,17 @@ - (void)playbackController:(id<BCOVPlaybackController>)controller playbackSessio
...
 
-#pragma mark IMAWebOpenerDelegate Methods
+#pragma mark - IMAPlaybackSessionDelegate Methods
+
+- (void)willCallIMAAdsLoaderRequestAdsWithRequest:(IMAAdsRequest *)adsRequest forPosition:(NSTimeInterval)position
+{
+    // for demo purposes, increase the VAST ad load timeout.
+    adsRequest.vastLoadTimeout = 3000.;
+    NSLog(@"ViewController Debug - IMAAdsRequest.vastLoadTimeout set to %.1f milliseconds.", adsRequest.vastLoadTimeout);
+
```
Comment (sjbushell) [inline IMA/FairPlayIMAPlayer/objc/FairPlayIMAPlayer/ViewController.m]: extra line?

**Comment by sjbushell** [comment]
👍

File: IMA/BasicIMAPlayer/objc/BasicIMAPlayer/ViewController.m (hunk 1 above)
Comment (timothyrodgers) [reply IMA/BasicIMAPlayer/objc/BasicIMAPlayer/ViewController.m]: √

File: IMA/FairPlayIMAPlayer/objc/FairPlayIMAPlayer/ViewController.m (hunk 2 above)
Comment (timothyrodgers) [reply IMA/FairPlayIMAPlayer/objc/FairPlayIMAPlayer/ViewController.m]: 👌

**Pull request merged**